docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
       - --entrypoints.appwrite_websecure.address=:443
       - --accesslog=true
     ports:
-      - 80:80
+      - 8080:80
```

Comment:
This is for gitpod right ? Is this the recommended way to do it ?

Reply:
Oh, i forgot to remove this. Yeah, its for GitPod. We cant use ports below 1024 (system ports)